neo/SmartContract/InteropService.cs

```diff
@@ -145,6 +145,8 @@ internal static bool CheckWitness(ApplicationEngine engine, UInt160 hash)
         {
             if (engine.ScriptContainer is Transaction tx)
             {
+                var ret = false;
+
                 foreach (var attribute in tx.Attributes)
```

Comment:
I think this works already for multiple attributes and same ScriptHash, right?
If it passes one, then it's fine I suppose.

Reply:
@erikzhang this probably resolves the situation of multiple Custom Hashes

Reply:
We can do a specific unit test for this

Reply:
Ok, double cosigners work fine. I had to adapt a little bit more the verification at CheckWitness.
Now, the wrong scripts are correctly faulting, during Application testmode (on "transfer", which cannot pass due to the lack of a valid witness).